docs/data/date-pickers/shortcuts/shortcuts.md

```diff
@@ -0,0 +1,60 @@
+---
+product: date-pickers
+title: Date and Time pickers - Shortcuts
+---
+
+# Shortcuts
+
+<p class="description">The date picker lets you add custom shortcuts.</p>
+
+## Adding shortcuts
+
+By default, pickers use the `PickersShortcuts` component to display shortcuts.
+This component accepts a `shortcurs` prop as an array of `PickersShortcutsItem`.
+Those items are made of two required attributes:
+
+- `label`: The string displayed on the shortcut chip. This property must be unique.
+- `getValue`: A function that returns the value associated to the shortcut.
+
+You can use `componentsProps.shortcus` to customize this props. For example to add a shortcut to Christmas Day, you can do the following:
+
+```jsx
+<DatePicker
+  componentsProps={{
+    shortcuts: {
+      shortcuts: [
+        {
+          label: 'Chrismas',
+          getValue: () => {
+            return dayjs(new Date(2023, 11, 25));
+          },
+        },
+      ],
+    },
+  }}
+/>
+```
+
+{{"demo": "BasicShortcuts.js", "bg": "inline"}}
+
+## Disabled dates
+
+By default, the shortcuts are disabled if the returned value does not pass validation.
+Here is an example where `minDate` is set to the middle of the year.
+
+{{"demo": "DisabledDatesShortcuts.js", "bg": "inline"}}
+
+## Range shortcuts [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
+
+Shortcuts on range pickers require `getValue` property to return an array with two values.
+
+{{"demo": "BasicRangeShortcuts.js", "bg": "inline"}}
```

Comment:
The "short version" of this demo code doesn't show getValue with two values. It's not a big deal, but it may make sense to display the relevant code regarding this difference in setup.

<img width="332" alt="image" src="https://user-images.githubusercontent.com/550141/213657937-8e6db31c-72d0-4aee-9b4b-080e3cb3ec79.png">